internal/integration/generate.go

```diff
@@ -34,6 +34,7 @@ package integration
 //
 //go:generate go run ../../cmd/ogen -v --clean -target test_enum_naming       ../../_testdata/positive/enum_naming.yml
 //go:generate go run ../../cmd/ogen -v --clean -target test_naming_extensions ../../_testdata/positive/naming_extensions.json
+//go:generate go run ../../cmd/ogen -v --clean -target test_type_extension ../../_testdata/positive/type_extension.yml
```

Comment:
It would be nice to have a JSON decode-encode-decode test too.

Reply:
Have added that for both JSON and params.

Reply:
Actually the tests for the params only cover decoding, not encoding. Shall I write E2E tests to cover everything?